### ISSUE TITLE
Feature request: VSCode code hint: function hint, parameter list hintu

### ISSUE THREAD

**Issue by dpy013** [nvaccess/nvda#10628]
hi
Current NVDA cannot report while using vscode VSCode code hint: function hint, parameter list hint
### Is your feature request related to a problem? Please describe.
Integration in NVDA
VSCode code hint: function hint, parameter list hint
Function
### Describe the solution you'd like
none
### Describe alternatives you've considered
none
### Additional context
open Visual Studio Code
Change language to html
Entering the following code will display the VSCode code hint: function hint, parameter list hint
!DOCTYPE read more... (Ctrl+Space)
dialog read more... (Ctrl+Space)
!DOCTYPE read more... (Ctrl+Space)
After entering the number code, a selection list is displayed, and the up and down arrows can select elements

**Comment by Adriani90** [comment]
cc: @pawelurbanski, @leonardder

**Comment by Adriani90** [comment]
@dingpengyu you can also use the VS Code addon for NVDA from here:
https://github.com/pawelurbanski/nvda-for-vs-code

**Comment by leonardder** [comment]
I think screen reader support for this should be implemented in Code itself. They are pretty a11y minded, so there might already be a feature request for this. cc @zersiax

**Comment by zersiax** [comment]
I know for a fact that in for example JavaScript, NVDA can already report things like "String, message" when typing , for example, Console.log followed by the opening bracket. 
This is not at all very well documented and I only know this because I was involved in making the initial autocomplete implementation work, but you can press ctrl+space when an autocomplete suggestion is reported to get what I think is the information you get when you hover your mouse. Generally the function name, its parameters, its description and its return parameters, akin to a JsDoc or JavaDoc comment. Is this in the accessibility docs?  cc @isidorn

**Comment by isidorn** [comment]
@dingpengyu looking at our VS Code repository it seems like this issue was solved here https://github.com/microsoft/vscode/issues/3851
However there might have been a regression. So I would please ask you to file a [new issue](https://github.com/microsoft/vscode/issues/new/choose) in the VS Code repository and ping me on the issue and then I will investigate more. Thanks a lot!
@zersiax thanks for looping me in.

**Comment by dingpengyu** [comment]
hi @isidorn 
I will use NVDA for related tests later
thanks

**Comment by cary-rowen** [comment]
@dpy013 
Is there any update on this issue?
After all this time, is VS Code currently behaving as you expected?

**Comment by Adriani90** [comment]
There is no update upon request from the initial author since more than 4 years, I am closing as abandoned. If you are still having this issue, please comment and we can reopen.
